doc/user/interop.md

```diff
@@ -113,42 +113,46 @@ If the object is a Ruby `Array` and the key is an integer:
 
 `READABLE` will be set if the index is in bounds (0 <= index < array.size)
 
-`WRITABLE` will be set if the index is in bound and the array is not frozen.
+`INSERTABLE` and `MODIFIABLE` will be set if the index is in bound and the array
+is not frozen.
 
 If the object is a Ruby `Hash`:
 
 `READABLE` will be set if the key is found.
 
-`WRITABLE` will be set if the key is found and the hash is not frozen.
+`INSERTABLE` will be set if the hash is not frozen.
 
-`INTERNAL` will never be set.
+`MODIFIABLE` will be set if the key is found and the hash is not frozen.
 
 Otherwise:
 
 `READABLE` will be set if the object responds to a method of the same name.
 
-`WRITABLE` will be set if the object responds to a method of the same name
+`MODIFIABLE` will be set if the object responds to a method of the same name
 appended with `=` and the object is not frozen.
 
+`INSERTABLE` will be false.
+
 If they key has a leading `@`:
```

Comment:
If `the` key

Reply:
Fixed in #1217 (but wasn't actually introduced in this PR anyway).